sdk/textanalytics/azure-ai-textanalytics/tests/test_recognize_pii_entities.py

```diff
@@ -596,6 +597,32 @@ def test_phi_domain_filter(self, client):
         self.assertEqual(phone.category, "PhoneNumber")
         self.assertEqual(microsoft.category, "Organization")
 
+    @GlobalTextAnalyticsAccountPreparer()
+    @TextAnalyticsClientPreparer()
+    def test_categories_filter(self, client):
+        result = client.recognize_pii_entities(
+            ["My name is Inigo Montoya, my SSN in 243-56-0987 and my phone number is 333-3333."],
+            categories_filter=[PiiEntityCategoryType.US_SOCIAL_SECURITY_NUMBER]
```

Comment:
super nitpicky: can you also make a call before this call in the same test with no categories filter and assert that there's 3? not absolutely required, but may be nice

Reply:
Updated!